clang/docs/checkedc/3C/INSTALL.md

```diff
@@ -14,14 +14,14 @@ Assuming you have already cloned the `checkedc-clang` repository, run the follow
 
 ```
 # See https://github.com/correctcomputation/checkedc-clang/issues/296 about this.
-(llvm/projects/checkedc-wrapper && git clone https://github.com/microsoft/checkedc)
+(cd llvm/projects/checkedc-wrapper && git clone https://github.com/microsoft/checkedc)
 
 mkdir build && cd build
```

Comment:
Are you putting the `build` directory in `checkedc-wrapper` ? How will this work with CLion?

Reply:
Or, maybe the parentheses mean that you are forking a sub-shell and not changing the CWD? Fancy.

Reply:
My understanding is that the repository just has to exist in that place and the build system will pick it up with the standard location of the `build` directory; I will test this later.  I don't know anything about CLion.  Do you want to have someone else test it?

Reply:
To be clear: Which directory will `build` end up in, with these instructions?

Reply:
Whoops, I didn't see your second comment before writing my first.  Yes, this is using a subshell and the `build` directory will end up in the usual place. :)

Reply:
OK then I think this is fine.